[observability] [apm] Document additional privileges for tail-based sampling (#1065)
elastic/docs-content: solutions/observability/apm/create-assign-feature-roles-to-apm-server-users.md

::::{note}
Assign additional APM feature roles to users as needed including the [*Writer role*](#apm-privileges-to-publish-events) and [*Central configuration management role*](#apm-central-config-role-note), both of which are required in most cases.
::::

The previous privileges should be sufficient for RUM source mapping to work properly as long as APM Server communicates with {{es}} successfully. If it fails, it may fallback to read source maps through {{kib}} if configured, which requires additional {{kib}} privileges. For more details, refer to the [{{stack}}](https://www.elastic.co/docs/api/doc/kibana/group/endpoint-apm-sourcemaps) or [{{serverless-short}}](https://www.elastic.co/docs/api/doc/serverless/group/endpoint-apm-sourcemaps) API documentation.

## Create a *tail-based sampling* role [apm-privileges-tail-based-sampling]

If [tail-based sampling](/solutions/observability/apm/tail-based-sampling.md) is enabled, the user will need additional privileges.

> To grant an APM Server user with the required privileges for reading tail-based sampling indices from {{es}}, assign the user the following privileges:

Reviewer: ```suggestion
APM Server users need the following privileges to read tail-based sampling indices from {{es}}:
```
